docs/source/testing.rst

```diff
@@ -97,41 +97,112 @@ Smoke test:
-```
-======================================================================
-FAIL: test_delete_single_provider_skips_delete_archived_data_if_customer_is_none (api.provider.test.tests_models.ProviderModelTest)
-Assert the delete_archived_data task is not called if Customer is None.
-----------------------------------------------------------------------
-Traceback (most recent call last):
-  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/mock.py", line 1209, in patched
+If you observe the following error in the tox tests, you may sometimes ignore it, due to tox not setting DEBUG=TRUE, to fix this you can export the variable to be true::
+    
+    FAIL: test_delete_single_provider_skips_delete_archived_data_if_customer_is_none 
+    (api.provider.test.tests_models.ProviderModelTest)
+    Assert the delete_archived_data task is not called if Customer is None.
+    ----------------------------------------------------------------------
+    Traceback (most recent call last):
+    File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/mock.py", line 1209, in patched
     return func(*args, **keywargs)
-  File "/Users/nbonilla/Documents/Koku/koku/koku/api/provider/test/tests_models.py", line 43, in test_delete_single_provider_skips_delete_archived_data_if_customer_is_none
+    File "/Users/nbonilla/Documents/Koku/koku/koku/api/provider/test/tests_models.py", line 43, in test_delete_single_provider_skips_delete_archived_data_if_customer_is_none
     self.aws_provider.delete()
-AssertionError: no logs of level WARNING or higher triggered on api.provider.models
-
-----------------------------------------------------------------------
-Ran 756 tests in 2025.258s
-FAILED (failures=1)
-```
-
-# Gotchas:
-- The logger is disabled by default during unit tests. If you are building a unit test that asserts that a log occurs, you must re-enable the logger. For example:
-
-```
-import logging
-with self.assertLogs(logger='masu.external.downloader.aws.aws_report_downloader', level='WARN') as cm:
-            logging.disable(logging.NOTSET)
-            self.aws_report_downloader._remove_manifest_file("None")
-            self.assertEqual(['WARN: Could not delete manifest file at'], cm.output)
-```
+    AssertionError: no logs of level WARNING or higher triggered on api.provider.models
+    
+
+==========
+PDB in IQE
+==========
+Begin a shell session into the docker container that runs IQE by running the following command::
+
+    koku/testing/run_test.sh bash
+
+since IQE itself is installed in this container, you may run the following command while inside the container to run a specific test::
+
+    iqe tests plugin hccm -k test_api_aws_storage_filtered_top --pdb
+
+or to run all tests::
+
+    iqe tests plugin hccm --pdb
+
+Any test that fails should start a PDB session.
+
```

Comment:
This section is in this file twice. Let's remove this section and leave the one down below.

```suggestion
```